src/config/env/local.json

```diff
@@ -33,5 +33,7 @@
   "GATSBY_SUBMISSION_THRESHOLD_DRAFT": "100",
   "GATSBY_SUBMISSION_THRESHOLD_GOVERNANCE": "100",
   "GATSBY_SEGMENT_KEY": "LOCAL_ENVIRONMENT_VARIABLE",
-  "GATSBY_LOCAL_ENV_VAR": "only for testing local"
+  "GATSBY_LOCAL_ENV_VAR": "only for testing local",
+  "DISCORD_SERVICE_ENABLED": "false",
```

Comment:
This variable is not necessary here as the discord service is only used server side 🤔